kep/9-capacity-scheduling/README.md

```diff
@@ -320,6 +321,233 @@ preemptor’s priority as potential victims in a node.
+
+#### 2. First Come First Serve Rule for namespace resources over its elastic quota min
+During the postFilter plugin / Preemption, if running current pod will lead to exceed of its current namespace min, the scheduler can only preempt lower priority pods in preemptor pod namespace. However, it cannot preempt other lower priority pods from other namespaces which are overusing their namespace resource min because the scheduler follows first come first serve rule.
+
+Below is a simple example. 
+##### Elastic Quota Configuration for namespace 1
+```
+apiVersion: v1
+kind: ElasticQuota
+metadata:
+  name: quota1
+  namespace: quota1
+spec:
+  max:
+    cpu: 2
+  min:
+    cpu: 0
+```
+##### Elastic Quota Configuration for namespace 2
+```
+apiVersion: v1
+kind: ElasticQuota
+metadata:
+  name: quota2
+  namespace: quota2
+spec:
+  max:
+    cpu: 2
+  min:
+    cpu: 0
+```
+##### Elastic Quota Configuration for namespace 3
+```
+apiVersion: v1
+kind: ElasticQuota
+metadata:
+  name: quota3
+  namespace: quota3
+spec:
+  max:
+    cpu: 2
+  min:
+    cpu: 1
+```
+#### Deployment on Namespace quota1
+```
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+  namespace: quota1
+  labels:
+    app: nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      name: nginx
+      labels:
+        app: nginx
+    spec:
+      schedulerName: capacityscheduler
+      containers:
+        - name: nginx
+          image: nginx
+          resources:
+            limits:
+              cpu: 1
+            `requests:`
+              cpu: 1
+```
+The nginx pod is able to run because it will borrow the 1 free cpu min from namespace quota3. Note, by default if the PriorityClassName is not configured, the pod will have a priority of 0.
+
+#### A sample Priority Class Deployment
+```
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: high-priority
+value: 1000000
+globalDefault: false
+description: "Sample High Priority Class"
+```
+
+#### Deployment on Namespace quota2
+```
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+  namespace: quota2
+  labels:
+    app: nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      name: nginx
+      labels:
+        app: nginx
+    spec:
+      schedulerName: capacityscheduler
+      priorityClassName: high-priority
+      containers:
+        - name: nginx
+          image: nginx
+          resources:
+            limits:
+              cpu: 1
+            `requests:`
+              cpu: 1
+```
+In this case, the high priority pod in namespace quota2 cannot run even though it has a higher priority compared to the running pod in namespace quota1 because the capacity scheduler follows first come first serve rule for pods overusing their namespace resource min. 
```

Comment:
```suggestion
In this case, the high priority pod in namespace quota2 cannot run even though it has a higher priority compared to the running pod in namespace quota1 because the capacity scheduler follows a restricted "in-namespace" preemption rule - pods within quota2 have overused their namespace resource min. 
```